CMakeLists.txt

```diff
@@ -11,6 +11,8 @@
 cmake_minimum_required(VERSION 3.13)
 project(PrusaSlicer)
 
+set(CMAKE_POLICY_VERSION_MINIMUM 3.5)
```

Comment:
Why not set the minimum cmake requirement to 3.10 everywhere? I don't think 3.5 is really required.

Reply:
I've just checked, even RHEL8 has at least cmake 3.11.

Reply:
Newer cmake versions will only support 3.10 policy as support for compatibility < 3.10 will be removed!

```
CMake Deprecation Warning at CMakeLists.txt:2 (cmake_minimum_required):
  Compatibility with CMake < 3.10 will be removed from a future version of
  CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.
```

Reply:
Worth noting that `CMAKE_POLICY_VERSION_MINIMUM` is not recognized in CMake < 4.0. But overall it's fine since it just effectively `cmake_minimum_required(VERSION X.XX...${CMAKE_POLICY_VERSION_MINIMUM}` and as long as CMake 4.0 is tested, then it's all good. Ideally the dependencies would be patched to specify a high higher bound themselves. but this is an ok intermediate solution